gulpfile.js

```diff
@@ -215,7 +220,18 @@ gulp.task('build-frontend-games-html', ['clean-frontend'], function () {
 
 gulp.task('build-frontend-js-vendor', ['clean-frontend'], function () {
   
-  return gulp.src(buildConfig.frontend.js.vendor.src)
+  var b = browserify({
+    entries: buildConfig.frontend.js.vendor.src,
+    debug: true
+  });
+
+  return b.bundle()
+    .pipe(source(buildConfig.frontend.js.vendor.src))
```

Comment:
It looks like this `source` function here defines what the name of the output file will be.

This seems to work:
```javascript
gulp.task('build-frontend-js-vendor', ['clean-frontend'], function () {
  
  var b = browserify({
    entries: buildConfig.frontend.js.vendor.src,
    debug: true
  });

  return b.bundle()
    .pipe(source('vendor.js'))
    .pipe(buffer())
    .pipe(sourcemaps.init({loadMaps: true}))
      .on('error', util.log)
    .pipe(sourcemaps.write('./'))
    .pipe(gulp.dest(buildConfig.frontend.js.vendor.dest));

});
```

Also, I want to keep `uglify` out for now, but when put it in, we need to make it optional, so that we can build with full unminified source for debugging.